text/0308-cli-advisories.md

```diff
@@ -0,0 +1,263 @@
+suppress them (per execution or indefinitely), they will use that and miss new
+advisories.
+
+### What future improvements can we make?
+
+**Resource filter**: Ideally, advisories should be as targeted as possible. The
+version filter addresses part of that requirement, but we can make it better by
+also adding a resource filter: if an issue only affects `FargateTaskDefinition`
+resources, for example, but this resource is not present in any of the stacks,
+the customer doesn’t need to see that particular warning.
+
+**Pipeline awareness**: Initially, we won’t treat the pipeline as a special
+case. By default, the advisories will be fetched and displayed and we expect
+users to never look at them. This behavior can always be overridden by
+suppressing the advisories. In future releases, the CLI will use the information
+about whether it’s running on a pipeline and block the deployment in case there
+is any advisory.
+
+## Appendix
+
+### Detailed design
+
+There are three parts to this solution: the data format, the GitHub workflow and
+the CLI logic.
+
+#### Data format
+
+We will use GitHub issues to publish and manage advisories. To be considered a
+valid advisory, the body of a GitHub issue must have the following fields:
+
+| Field                  | Description                                                                                                                                                | Format                                 | Mandatory? |
+| ---------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- | ---------- |
+| Summary                | A paragraph with more information                                                                                                                          | Free-form text                         | Yes        |
+| Affected component     | The CLI or the construct library (or both)                                                                                                                 | A non-empty subset of {CLI, Framework} | Yes        |
+| Affected version range | Version range using the semver format                                                                                                                      | semver                                 | No         |
+| Suggested fix          | An action the user can take to mitigate the problem. If it requires a longer explanation, this field should be omitted in favor of the resource linked to. | Free-form text                         | No         |
+
+In order to keep the issue human readable as well as easily parseable, each of
+these fields should be indicated by a Markdown header (`### Summary`, `###
+Affected component` etc). The piece of text following the header will be
+interpreted as the content of the field.
+
```

Comment:
Add an example

Reply:
I just added a reference to the Working backwards section.